platform/platform-app/src/main/resources/features.xml

```diff
@@ -37,7 +37,7 @@
     <!--START Camel Features-->
     <!-- DO NOT EDIT
         All feature definitions come from 3rd party feature file:
-        <repository>mvn:org.apache.camel.karaf/apache-camel/2.19.0/xml/features</repository>
+        <repository>mvn:org.apache.camel.karaf/apache-camel/2.19.4/xml/features</repository>
```

Comment:
❓ Would it make sense to use the `camel.version` property defined in the root pom?

Reply:
Sure, I am okay with that.

Reply:
Fixed.

Reply:
❗ I don't think we should use `${camel.version}` or `${cxf.version}` for any of these code comments. I want it to be very clear exactly what version the properties and features were copied from. I think it protects against people upgrading cxf without upgrading the properties and features.

Reply:
@emmberk  I feel like it's very easy to overlook that version number, and it adds another line of code we need to update every time we upgrade cxf or camel, however, I agree that we should very clear what versions the properties and features come from to help prevent making mistakes when doing upgrades.

Reply:
Reverted.